packages/server/postgres/getPostgres.ts

```diff
@@ -0,0 +1,11 @@
+import { Pool } from 'pg'
+
+let pool
+const getPostgres = () => {
+  if (!pool) {
+    pool = new Pool({ max: 1 })
```

Comment:
we'll wanna step this up for production. i think for rethinkdb we have 50 conns in prod & 3 in dev.